docs/tutorials/pytorch-mnist-local-qs.rst

```diff
@@ -63,6 +63,8 @@ To start your experiment, you'll need a Determined cluster. If you are new to De
    # If your machine does not have GPUs:
    det deploy local cluster-up --no-gpu
 
+.. include:: ../_shared/pip-install-determined.txt
```

Comment:
Thoughts about also including the note in https://docs.determined.ai/latest/quickstart-mdldev.html?

Reply:
added